src/main/java/com/technicjelle/bluemapofflineplayermarkers/Config.java

```diff
@@ -4,28 +4,34 @@
 import java.net.URL;
 
 public class Config {
-	public final String markerSetId = "offplrs";
+
+	private final Main plugin;
+
+	public String markerSetId;
 	public String markerSetName;
 	public boolean useBlueMapSource;
 	public URL skinURL;
 	public boolean verboseErrors;
 
-	public Config() {
+	public Config(Main plugin) {
+		this.plugin = plugin;
+		plugin.reloadConfig();
 		loadConfig();
 	}
 
 	private void loadConfig() {
-		markerSetName = "Offline Players"; //TODO: https://github.com/TechnicJelle/BlueMapOfflinePlayerMarkers/issues/10
-		useBlueMapSource = true;
+		markerSetId = plugin.getConfig().getString("MarkerSetId");
```

Comment:
I don't think users of this plugin need to be able to change the MarkerSet ID. It's [so well hidden](https://discord.com/channels/665868367416131594/751804128749027421/1033518402209329192) in the BlueMap interface that basically no-one would ever see it anyway.
Removing this from the config would be better, in my opinion.
That takes away an option in the config for users to be confused about too.

Reply:
Of course the MarkerSet Name should stay in the config.